docs/markdown/Machine-files.md

```diff
@@ -155,6 +153,13 @@ case). For compilers like MSVC and Clang-Cl, this is the path to a
 linker for Meson to invoke, such as `link.exe` or `lld-link.exe`.
 Support for `ld` is *new in 0.53.0*
 
+These values must be either strings or an array of strings. Previous to Meson
+1.3.0, if a path contains spaces, it must be wrapped into an array:
+
+```ini
+c = ['C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.34.31933/bin/Hostx64/x64/cl.exe']
```

Comment:
Should you throw a comment above this that says `# Prior to 1.3.0`. Then does it make sense to have an after example too?

Reply:
I don't understand the meaning of this section at all, actually. It appears to be an incorrect editorialization?